cmd/dotmesh-server/controller.go

```diff
@@ -269,19 +270,19 @@ func (s *InMemoryState) getOne(ctx context.Context, fs string) (DotmeshVolume, e
 		sort.Sort(ByAddress(servers))
 
 		if ok {
-
 			for _, server := range servers {
 				numSnapshots := len(fsm.GetSnapshots(server.Id))
 				state := fsm.GetMetadata(server.Id)
```

Comment:
What else is likely to come up in metadata? Would be nicer to have it be a struct